emmet-core/emmet/core/qchem/molecule.py

```diff
@@ -252,4 +300,29 @@ def construct_deprecated_molecule(
             task_types=task_types,
             deprecated=deprecated,
             deprecated_tasks=deprecated_tasks,
-        )
\ No newline at end of file
+        )
+
+
+def best_lot(mol_doc: MoleculeDoc,
+             funct_scores: Dict[str, int] = SETTINGS.QCHEM_FUNCTIONAL_QUALITY_SCORES,
+             basis_scores: Dict[str, int] = SETTINGS.QCHEM_BASIS_QUALITY_SCORES,
+             solvent_scores: Dict[str, int] = SETTINGS.QCHEM_SOLVENT_MODEL_QUALITY_SCORES
+             ) -> LevelOfTheory:
+    """
+
+    Return the best level of theory used within a MoleculeDoc
+
+    :param mol_doc: MoleculeDoc
```

Comment:
Could these docstrings be slightly more explicit, e.g. add "Scores are integers where higher values indicate a higher preference or quality"